frontend/src/components/atoms/SelectField/SelectField.js

```diff
@@ -35,9 +37,11 @@ class SelectField extends Component {
 }
 
 SelectField.propTypes = {
+  // eslint-disable-next-line react/require-default-props
```

Comment:
Rather than disabling this warning, you could just add the default props like this:

```
SelectField.defaultProps = {
    mode: "multiple",
    ....
};
```

https://stackoverflow.com/questions/57009097/eslint-react-prop-types-vs-react-require-default-props

Reply:
Since Ant Design's Select component does not support default mode (see 'mode' API https://ant.design/components/select/), after a discussion and an additional exploration a decision has been made to keep the warning disabled.